tests/testthat/test-DataWrangling.R

```diff
@@ -107,6 +107,70 @@ test_that("FindExpectedReferenceTreatment() returns first matching treatment whe
   expect_equal(!!result, "Placebo")
 })
 
+test_that(".FixColumnNameCases() fixes cases for continuous long data", {
+  data <- CleanData(read.csv("Cont_long.csv"))
+  names(data) <- c("sTuDy", "t", "n", "mEaN", "sD")
+  allowed_names = c("Study", "T", "N", "Mean", "SD")
+  
+  expect(all(!names(data) %in% allowed_names), failure_message = "Column names were not setup for the test correctly.")
+  
```

Comment:
The ! should be in front of all( ). The same problem in several other places

Reply:
I think it is correct as it is. I am asserting that all of the names are not in `allowed_names`, not that at least one isn't.

Reply:
I can see where you're coming from.